unime/src/app.css

```diff
@@ -3,148 +3,189 @@
 @tailwind utilities;
 
 @layer base {
-  /* @font-face { */
-  /* font-family: 'Satoshi-Variable';
-    src: url('/fonts/Satoshi-Variable.woff2') format('woff2'); */
-  /* font-family: 'Inter-Variable';
-    src: url('/fonts/Inter-Variable.ttf') format('truetype'); */
-  /* font-family: 'Inter'; */
-  /* src: url('/fonts/Inter-VariableFont_slnt,wght.ttf') format('truetype'); */
-  /* src: url('/fonts/Inter.var.woff2') format('woff2'); */
-  /* } */
-
+  /* Font: Inter. */
   @font-face {
     font-family: 'Inter';
-    src: url('/fonts/Inter/static/Inter-Thin.ttf') format('truetype');
+    font-display: swap;
+    font-style: normal;
     font-weight: 100;
+    src: url('./fonts/inter/inter-v18-latin-100.woff2') format('woff2');
   }
 
+  /* @font-face {
```

Comment:
Since they are commented out, they don't add any weight to the app. But I would leave them around because if we remove them now and someone wants something in italic, it will be a pain to figure again where to get them from.

Reply:
Okay 👍